editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2665,11 +2667,13 @@ void CanvasItemEditor::_update_lock_and_group_button() {
 	}
 
 	lock_button->set_visible(!all_locked);
-	lock_button->set_disabled(selection.is_empty());
+	lock_button->set_disabled(!has_canvas_item || selection.is_empty());
```

Comment:
```suggestion
	lock_button->set_disabled(!has_canvas_item);
```
If selection is empty, `has_canvas_item` will always be false.

Reply:
Thanks！